.github/workflows/main.yml

```diff
@@ -0,0 +1,15 @@
+name: CI
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+    - name: Run tests and attempt building
+      run: |
+        export PATH=$(go env GOPATH)/bin:$PATH
+        golangci-lint run .
```

Comment:
we need to make sure `golangci-lint` is in the container which doesn't seem to be the case now

Reply:
I'm trying to use this action now:
https://github.com/marketplace/actions/golangci-lint